datafusion/expr/src/udaf.rs

```diff
@@ -635,6 +635,14 @@ pub trait AggregateUDFImpl: Debug + Send + Sync {
     fn documentation(&self) -> Option<&Documentation> {
         None
     }
+
+    /// Indicates whether the aggregation function is monotonic as a set function. A set
+    /// function is monotonically increasing if its value increases as its argument grows
+    /// (as a set). Formally, `f` is a monotonically increasing set function if `f(S) >= f(T)`
```

Comment:
I think it would also help to copy the formal definition on to `AggregateExprSetMonotonicity` as well to make it more likely someone will find this